Userland/Utilities/tail.cpp

```diff
@@ -71,12 +72,9 @@ static off_t find_seek_pos(Core::File& file, int wanted_lines)
     return pos;
 }
 
-int main(int argc, char* argv[])
+ErrorOr<int> serenity_main(Main::Arguments args)
```

Comment:
The vast majority of the code base seems to refer to this as `arguments` so lets try to standardize on that, please fixup all commits in this PR to follow suite? 

```bash
$ rg -I Main::Arguments | rg serenity_main| sort | uniq -c

      1 ErrorOr<int> serenity_main(Main::Arguments argmuments)
      9 ErrorOr<int> serenity_main(Main::Arguments args)
    174 ErrorOr<int> serenity_main(Main::Arguments arguments)
      1 ErrorOr<int> serenity_main(Main::Arguments main_arguments)
     31 ErrorOr<int> serenity_main(Main::Arguments)
      1 ErrorOr<int> serenity_main(Main::Arguments);
      2 ErrorOr<int> serenity_main([[maybe_unused]] Main::Arguments arguments)
```